src/components/Contacts/ContactDetails/ContactDetailsTab/Tags/ContactTags.test.tsx

```diff
@@ -41,11 +43,10 @@ describe('ContactTags', () => {
   });
 
   it('should add a tag', async () => {
-    const cache = new InMemoryCache();
+    const cache = new InMemoryCache({ addTypename: false });
```

Comment:
I have this test now set up very similar to the commentList form test that I updated in this commit. Still not sure why it won't run as expected.